packages/core/components/FileDetails/index.tsx

```diff
@@ -99,7 +99,8 @@ export default function FileDetails(props: Props) {
             if (isOnWeb && isZarr) {
                 if (fileDetails.size && fileDetails.size > 0) {
                     setCalculatedSize(fileDetails.size);
-                } else {
+                } else if (fileDownloadService.isS3Url(fileDetails.path)) {
+                    // Currently unable to calculate file size for local, non-s3 zarr files
```

Comment:
I think we could support non-s3 files, we would just need a way to list their subfiles. Local we cannot do from web since we need permission for every subfile (>10,000 in some cases)

Reply:
>Local we cannot do from web since we need permission for every subfile (>10,000 in some cases)

Can you re-phrase this? I don't think I understand, are you saying locally (i.e. on desktop?) we can't get files from S3 files?

Reply:
I was trying to say that I think we could support other cloud locations from web (ex. Google/Microsoft) I just haven't seen any examples.

The second sentence is saying we cannot do local access for web because of the multifile structure and how web browsers allow access to files.